bin/wp-enforcer

```diff
@@ -18,6 +22,18 @@ error() {
   exit 1;
 }
 
+update_root_path() {
+
+  if [[ "global" -eq ${PHPCS_PATH} ]]; then
+    sed -i.bak s={phpcs_path}=phpcs=g ${PROJECT}/.git/hooks/pre-commit
+    sed -i.bak s={phpcs_standard}=${PHPCS_STANDARD}=g ${PROJECT}/.git/hooks/pre-commit
+  else
+    sed -i.bak s={phpcs_path}=${PROJECT}/vendor/bin/phpcs=g ${PROJECT}/.git/hooks/pre-commit
```

Comment:
I'm not sure it's a good idea to leave the `{phpcs_standard}` placeholder in the pre-commit hook file, even if it doesn't actually get executed in this scenario. Maybe moving line 29 out of the conditional with the project placeholder so it's always run?

Reply:
Yes this is more about the question before of it.

Reply:
Again, my concern is that we would be leaving what amounts to a syntax error in the script, even if we can be confident it won't actually run. I understand that replacing the placeholder in all cases doesn't alter the behavior in many cases, my concern is that we should maintain a certain standard of readability and clarity in the installed git hook script.

Reply:
So ... like phpcs_standard on top of all conditionals just to keep the script sanitize of any weird text that could cause? I'm right?